src/Symfony/Component/Console/Command/Command.php

```
@@ -284,6 +284,13 @@ public function setCode($code)
             throw new \InvalidArgumentException('Invalid callable provided to Command::setCode.');
         }
 
+        if (PHP_VERSION_ID >= 50400 && $code instanceof \Closure) {
+            $r = new \ReflectionFunction($code);
+            if (null === $r->getClosureThis()) {
```

when it's `null` should we throw an exception ?